ui/src/helpers/FormService.ts

```diff
@@ -0,0 +1,8 @@
+export class FormService{
```

Comment:
why is the name FormService?
I would replace this class with a simple function which checks isValidEmail